pr_agent/git_providers/github_provider.py

```diff
@@ -84,6 +84,9 @@ def _get_issue_handle(self, issue_url) -> Optional[Issue]:
                                    f"not have a valid repository: {self.get_git_repo_url(issue_url)}")
                 return None
             # else: Valid repo handle:
+            # Store the repository object for later use
+            self.repo_obj = repo_obj
+            self.repo = repo_name
```

Comment:
if we're going to start working around issues, we might want to have the `self.issue` field be saved as well